gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+networkTimeout=10000
```

Comment:
This is generated by gradle so I was opting not to change it. Can remove if you want, but you'd have to manually remove it every time the wrapper's updated

Reply:
Maybe we should hold off with the Gradle bump and make this separate: https://github.com/gradle/gradle/issues/23215

Reply:
We haven't noticed it in our project tbh

Reply:
If it's auto-generated then that's fine :+1:

I would agree on waiting to bump until that issue is resolved, we're seeing similar issues with a draft upgrade internally

Reply:
This isn't propagated to downstream users though and isn't affected in this project itself. Are you seeing issues on this project?

Reply:
Hmm without doing some testing I think it would be hard to say that we're not affected by it (unless you mean you compared the memory usage already), even if it's not causing any failures yet. E.g. memory usage could be inflated for us now but not quite hitting the point of getting OOMs yet. 

My thought process is: is there any particular improvement in 7.6 we need right now compared to waiting for 7.6.1 when this issue will be resolved? Open to being convinced to leave it but if we don't have a need for it right now, waiting seems preferable.

Reply:
[5c2aff3](https://github.com/square/anvil/pull/678/commits/5c2aff3a9b75894fe8065b162756de2b50a7ad18)